core/src/main/java/hudson/util/DoubleLaunchChecker.java

```diff
@@ -91,20 +80,29 @@ public DoubleLaunchChecker() {
         home = Jenkins.get().getRootDir();
     }
 
+    @Override
+    public String getDisplayName() {
+        return Messages.DoubleLaunchChecker_duplicate_jenkins_checker();
+    }
+
+    @Override
+    public boolean isActivated() {
+        return activated;
+    }
+
     protected void execute() {
+        LOGGER.fine("running detector");
         File timestampFile = new File(home, ".owner");
 
         long t = timestampFile.lastModified();
-        if (t != 0 && lastWriteTime != 0 && t != lastWriteTime && !ignore) {
+        if (t != 0 && lastWriteTime != 0 && t != lastWriteTime && isEnabled()) {
```

Comment:
Move the `isEnabled()` call above or in the `Schedule` class to avoid reading the file every time?

Reply:
This is run only every 75m, and even then only mattering in the rare case that this monitor was activated and then disabled, and we are still _writing_ the same file unconditionally, so should not make much difference. Going to leave it as is to keep diff smaller.